src/default.conf

```diff
@@ -0,0 +1,29 @@
+local show_directory
+readonly show_directory="#[fg=$thm_pink,bg=$thm_bg,nobold,nounderscore,noitalics]$r_left_separator#[fg=$thm_bg,bg=$thm_pink,nobold,nounderscore,noitalics]  #[fg=$thm_fg,bg=$thm_gray] #{b:pane_current_path} "
```

Comment:
The directory icon here seems to have an extra space compared to the other icons in this block.

Reply:
Good catch on this. Looks like it was also caught in #49 as well.